src/App.js

```diff
@@ -1,5 +1,98 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+
 class App {
-  async run() {}
+  async getCarNames(query) {
+    try {
+      const input = await Console.readLineAsync(query);
+      const carNames = input.split(',').map((name) => name.trim());
+      return carNames;
+    } catch (error) {
+      throw new Error(`[ERROR] ${error}`);
+    }
+  }
+
+  async getTryNumber(query) {
+    try {
+      const input = await Console.readLineAsync(query);
+      if (Number.isNaN(Number(input))) {
+        throw new Error(`[ERROR]`);
+      }
+      return input;
+    } catch (error) {
+      throw new Error(`[ERROR] ${error}`);
+    }
+  }
+
+  // 입력받은 경주용 자동차 이름을 {이름: 점수}의 점수판 객체로 만들기
+  setGame(input) {
+    const scoreBoard = {};
+    input.forEach((name) => {
+      scoreBoard[name] = 0;
+    });
+    return scoreBoard;
+  }
+
+  // 1회차 실행결과 계산하기
+  executionResult(input) {
+    const updatedInput = { ...input };
+
+    Object.keys(updatedInput).forEach((item) => {
+      const isFowrad = Random.pickNumberInRange(0, 9);
+      if (isFowrad >= 4) {
```

Comment:
`4`를 상수로 선언하여 해당 값이 어떤 의미를 담고 있는지 명확히 드러내면 좋을 것 같습니다! 
```suggestion
      if (isFowrad >= MIN_SCORE_TO_MOVE) {
```
참고로 이 부분은 2주 차 공통 피드백에 포함된 내용입니다 😊

Reply:
지적 감사합니다!